cli/flox-rust-sdk/src/providers/upgrade_checks.rs

```diff
@@ -194,12 +194,17 @@ fn read_upgrade_information(
     }
 
     let info_str =
-        fs::read_to_string(upgrade_information_path).map_err(UpgradeChecksError::Read)?;
+        fs::read_to_string(&upgrade_information_path).map_err(UpgradeChecksError::Read)?;
+
+    let info = match serde_json::from_str(&info_str) {
+        Ok(info) => Some(info),
+        Err(err) => {
+            debug!(?err, path = %upgrade_information_path.as_ref().display(), "discarding existing upgrade information");
```

Comment:
nb, but i wish we can be consistent with these eventually

```suggestion
            debug!(?err, path = ?upgrade_information_path.as_ref(), "discarding existing upgrade information");
```

Reply:
I looked at changing some of the others but that turns out to be a deep rabbit hole